### ISSUE TITLE
Android build error : duplicate class com.google.android.exoplayer

### ISSUE THREAD

**Issue by uchar** [AppLovin/AppLovin-MAX-Unity-Plugin#175]
In the latest version of Applovin SDK ( 5.4.3) after enabling MyTarget, I have the following build error :
``` 
> 1 exception was raised by workers:
  java.lang.RuntimeException: java.lang.RuntimeException: Duplicate class 
com.google.android.exoplayer2.ui.DownloadNotificationHelper found in modules jetified-com.google.android.exoplayer.exoplayer-core-2.16.1-runtime.jar 
(:com.google.android.exoplayer.exoplayer-core-2.16.1:) and jetified-com.google.android.exoplayer.exoplayer-ui-2.15.1-runtime.jar (:com.google.android.exoplayer.exoplayer-ui-2.15.1:) 
```

What should I do to fix this?

**Comment by uchar** [comment]
Maybe it's related to your android SDK's latest changes :

![image](https://user-images.githubusercontent.com/5172296/174746838-e61781be-b5da-40db-af7f-0c541acd4ca3.png)

**Comment by uchar** [comment]
For now, as a temporary solution, I enabled "Custom Main Gradle Template" and "Custom Gradle Template Properties" from Player setting 
And then add this under dependencies in "Assets/Plugins/Android/mainTemplate.gradle" : 
```
implementation("com.google.android.exoplayer:exoplayer-ui") {
    version {
        strictly("2.16.1")
      }
    }
```
However, it's probably not right to mess with the dependency versions like this because Applovin is not tested with that version of exoplayer and may cause unintended behaviors.

**Comment by filgreen3** [comment]
Hi @uchar, sorry for ping you, and thanks for your solution. Is it ok, to just delete older duplicated class? It seems no error, and app open as normal, but it feels bad. Could be any problem in app runtime ?

**Comment by uchar** [comment]
@filgreen3 Hi, It's probably same as what I do, By removing that, it use the newer version.
But the correct way would be when applovin address this issue in their update

**Comment by Kenorbrook** [comment]
+1

**Comment by aliayan15** [comment]
+1

**Comment by kobyle69** [comment]
Any update here?

**Comment by Lorenzo45** [comment]
Hi everyone, we've released SDK 11.5.0 (Unity Plugin 5.5.0) which removes the external ExoPlayer dependency from our SDK. Would you be able to test it and let us know if it fixes the build issues for you?

**Comment by uchar** [comment]
@Lorenzo45 Thanks, I test it
It's working fine.
